ndbench-cli/build.gradle

```diff
@@ -33,6 +36,13 @@ dependencies {
     compile project(':ndbench-janusgraph-plugins')
     compile project(':ndbench-sample-plugins')
     testCompile group: 'junit', name: 'junit', version: '4.12'
+    download {
```

Comment:
Do we need the DAX library in the CLI?

Reply:
for some reason because of the way we are downloading the dependency in ndbench-dynamodb, ndbench-cli does not have the library in class path when I start the CLI. so I added it in the same way here.